packages/gateway/src/graphql/schema.graphql

```diff
@@ -1,1512 +0,0 @@
-  id: ID!
-  type: String
-  deceasedName: [HumanName]
-  dateOfDeath: Date
-  registration: RegistrationSearchSet
-}
-
-type EventProgressData {
-  timeInProgress: Int
-  timeInReadyForReview: Int
-  timeInRequiresUpdates: Int
-  timeInWaitingForApproval: Int
-  timeInWaitingForBRIS: Int
-  timeInReadyToPrint: Int
-}
-
-type MesssageDescriptor {
-  id: String
-  description: String
-  defaultMessage: String
-}
-
-input IdentityInput {
-  id: ID
-  type: IdentityIDType
-  otherType: String
-}
-
-input ContactPointInput {
-  system: String
-  value: String
-  use: String
-}
-
-input AddressInput {
-  use: String
-  type: AddressType
-  text: String
-  line: [String]
-  city: String
-  district: String
-  state: String
-  postalCode: String
-  country: String
-  from: Date
-  to: Date
-}
-
-input AttachmentInput {
-  _fhirID: ID
-  contentType: String
-  data: String
-  status: String
-  originalFileName: String
-  systemFileName: String
-  type: AttachmentType
-  description: String
-  subject: AttachmentSubject
-  createdAt: Date
-}
-
-input DeceasedInput {
-  deceased: Boolean
-  deathDate: String
-}
-
-input RegWorkflowInput {
-  type: RegStatus
-  user: UserInput
-  timestamp: Date
-  reason: String
-  comments: [CommentInput]
-  location: LocationInput
-  timeLoggedMS: Int
-}
-
-input CertificateInput {
-  collector: RelatedPersonInput
-  hasShowedVerifiedDocument: Boolean
-  payments: [PaymentInput]
-  data: String
-}
-
-input CorrectionInput {
-  requester: ID
-  hasShowedVerifiedDocument: Boolean
-  attestedAndCopied: Boolean
-  noSupportingDocumentationRequired: Boolean
-  payments: [PaymentInput]
-  values: [CorrectionValueInput]
-  location: LocationInput
-  data: String
-  reason: String
-  note: String
-}
-
-input ReasonsNotApplyingInput {
-  primaryCaregiverType: PrimaryCaregiverType
-  reasonNotApplying: String
-  isDeceased: Boolean
-}
-
-type BirthFee {
-  ON_TIME: Float
-  LATE: Float
-  DELAYED: Float
-}
-
-type DeathFee {
-  ON_TIME: Float
-  DELAYED: Float
-}
-
-input BirthFeeInput {
-  ON_TIME: Float
-  LATE: Float
-  DELAYED: Float
-}
-
-input DeathFeeInput {
-  ON_TIME: Float
-  DELAYED: Float
-}
-
-input MesssageDescriptorInput {
-  id: String
-  description: String
-  defaultMessage: String
-}
-
-type Payment {
-  paymentId: ID
-  type: PaymentType
-  total: Float
-  amount: Float
-  outcome: PaymentOutcomeType
-  date: Date
-}
-
-enum PrimaryCaregiverType {
-  MOTHER_AND_FATHER
-  MOTHER
-  FATHER
-  INFORMANT
-  LEGAL_GUARDIAN
-  OTHER
-}
-
-input CommentInput {
-  user: UserInput
-  comment: String
-  createdAt: Date
-}
-
-input PaymentInput {
-  paymentId: ID
-  type: PaymentType
-  total: Float
-  amount: Float
-  outcome: PaymentOutcomeType
-  date: Date
-  data: String
-}
-
-input CorrectionValueInput {
-  section: String
-  fieldName: String
-  oldValue: String
-  newValue: String
-}
-
-enum PaymentType {
-  MANUAL
-}
-
-enum PaymentOutcomeType {
-  COMPLETED
-  ERROR
-  PARTIAL
-}
```

Comment:
That's lots of lines being deleted. What caused it to change so much?

Reply:
There was a problem with graphQL schema generation. Fixed now.